content/competitions/codejam.html

```diff
@@ -59,9 +55,12 @@ <h3>What do I do on the day?</h3>
                     href="https://www.youtube.com/c/UQComputingSociety">YouTube</a>, at the start and end of the day.
                 The actual coding competition will be hosted on <a href="https://www.hackerrank.com/">HackerRank</a>.
```

Comment:
The first rich text link to HackerRank should be far earlier on this page.

Reply:
Would linking in "So, how does it all work?" be better?